### PR TITLE
Gylych file created.TestGit2 updated.

### PR DESCRIPTION
**Pull request by Gylych1110**
Gylych PR

### CODE REVIEW

**Comment by Gylych1110** [comment]
hello

File: src/Test2/Gylych.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package Test2;
+
+public class Gylych {
+    public static void main(String[] args) {
+        System.out.println("Created locally in Gylych branch");
```
Comment (Ozleblebici) [inline src/Test2/Gylych.java:5]: please add extra test to log out functionality!!

Comment (Gylych1110) [reply src/Test2/Gylych.java:5]: how I can do?

Comment (Ozleblebici) [reply src/Test2/Gylych.java:5]: Go to your branch in IntelliJ add extra println for test1 then commit and push it !

**Review by Ozleblebici — APPROVED** [review]
nice job

**Review by Ozleblebici — APPROVED** [review]

**Pull request closed without merging**